src/js/cf-expandables.js

```diff
@@ -5,110 +5,255 @@
-      };
-
-      this.handleClick = function( ev ) {
-        ev.preventDefault();
-        ev.stopPropagation();
-        this.toggle();
-        if ( accordion ) {
-          $siblings.each( function( index, sibling ) {
-            sibling.collapse();
-          });
-        }
-      };
+    /**
+     * Initializes the initial state of the Expandable UI.
+     */
+    initUI: function initUI() {
+      var $el = this.$el;
 
-      this.toggle = function() {
-        if ( $target.attr('aria-pressed') === 'true' ) {
-          this.collapse();
-        } else {
-          this.expand();
-        }
-      };
-
-      this.expand = function( duration ) {
-        $cueOpen.css( 'display', 'none' );
-        $cueClose.css( 'display', 'inline' );
-        $content.attr( 'aria-expanded', 'true' );
-        $target.attr( 'aria-pressed', 'true' );
-        if ( typeof duration === 'undefined' ) {
-          duration = $.fn.expandable.calculateExpandDuration( $content.height() );
+      for ( var uiKey in this.ui ) {
+        if ( this.ui.hasOwnProperty( uiKey ) ) {
+          this['$' + uiKey] = $el.find( this.ui[uiKey] + ':first' );
         }
-        $this.addClass('expandable__expanded');
-        $content.slideDown({
-          duration: duration,
-          easing: 'easeOutExpo'
-        });
-      };
-
-      this.collapse = function( duration ) {
-        $cueOpen.css( 'display', 'inline' );
-        $cueClose.css( 'display', 'none' );
-        $content.attr( 'aria-expanded', 'false' );
-        $target.attr( 'aria-pressed', 'false' );
-        if ( typeof duration === 'undefined' ) {
-          duration = $.fn.expandable.calculateCollapseDuration( $content.height() );
-        }
-        $this.removeClass('expandable__expanded');
-        $content.slideUp({
-          duration: duration,
-          easing: 'easeOutExpo'
-        });
-      };
+      }
 
-      this.init();
+      this.$target.attr( 'aria-controls', this.$content.attr( 'id' ) );
 
-    });
+      if ( $el.hasClass( this.expandedClass ) ) {
+        this.expand( 0 );
+      } else {
+        this.collapse( 0 );
+      }
+    },
 
-  };
+    /**
+     * Initializes the Expandable dom events.
+     */
+    initEvents: function initEvents() {
+      this.$el.on( 'click', '.expandable_target',
+                   _throttle( this.throttleDuration, this.toggle, this )
+                 );
+    },
 
-  $.fn.expandable.calculateExpandDuration = function( height ) {
-    return $.fn.expandable.constrainValue( 450, 900, height * 4 );
-  };
+    /**
+     * Expands or collapses the Expandable.
+     *
+     * @param {object} event - jQuery event object.
+     */
+    toggle: function toggle( event ) {
+      event.preventDefault();
+      event.stopPropagation();
 
-  $.fn.expandable.calculateCollapseDuration = function( height ) {
-    return $.fn.expandable.constrainValue( 350, 900, height * 2 );
-  };
+      if ( this.isInAccordion ) {
```

Comment:
fixed in https://github.com/sebworks/cf-expandables/commit/4f9e0c59ac1aa93385c859ca507afcd135f72d75